Tools/c-analyzer/cpython/globals-to-fix.tsv

```diff
@@ -86,6 +86,7 @@ Objects/sliceobject.c	-	PyEllipsis_Type	-
 Objects/sliceobject.c	-	PySlice_Type	-
 Objects/tupleobject.c	-	PyTupleIter_Type	-
 Objects/tupleobject.c	-	PyTuple_Type	-
+Objects/typeobject.c    -   _PyBufferWrapper_Type   -
```

Comment:
```suggestion
Objects/typeobject.c	-	_PyBufferWrapper_Type	-
```
I used the tab key in my editor, but I suppose editors really don't want to output tabs. Thanks for catching!